test/lint/lint-circular-dependencies.sh

```diff
@@ -56,6 +56,7 @@ EXPECTED_CIRCULAR_DEPENDENCIES=(
     "qt/bitcoingui -> qt/guiutil -> qt/bitcoingui"
     "qt/guiutil -> qt/optionsdialog -> qt/guiutil"
     "qt/guiutil -> qt/qvalidatedlineedit -> qt/guiutil"
+    "qt/clientmodel -> qt/guiutil -> qt/walletmodel -> qt/clientmodel"
```

Comment:
just-a-note: this circular dependency due to missing bitcoin/bitcoin#17513

Reply:
Resolved in latest push